realm/realm-library/src/androidTestObjectServer/java/io/realm/SessionTests.java

```diff
@@ -130,12 +131,12 @@ public void onError(SyncSession session, ObjectServerError error) {
                         final ClientResetRequiredError handler = (ClientResetRequiredError) error;
                         String filePathFromError = handler.getOriginalFile().getAbsolutePath();
                         String filePathFromConfig = session.getConfiguration().getPath();
-                        System.out.println(">>>>>>>>>>>>>>>>> filePathFromError  : " + filePathFromError);
-                        System.out.println(">>>>>>>>>>>>>>>>> filePathFromConfig : " + filePathFromConfig);
-
-                        System.out.println(">>>>>>>>>>>>>>>>> handler.getBackupFile().exists()   : " + handler.getBackupFile().exists());
-                        System.out.println(">>>>>>>>>>>>>>>>> handler.getOriginalFile().exists() : " + handler.getOriginalFile().exists());
+                        Log.e("H4XOR", ">>>>>>>>>>>>>>>>> filePathFromError  : " + filePathFromError);
+                        Log.e("H4XOR", ">>>>>>>>>>>>>>>>> filePathFromConfig : " + filePathFromConfig);
 
+                        Log.e("H4XOR", ">>>>>>>>>>>>>>>>> handler.getBackupFile().exists()   : " + handler.getBackupFile().exists());
+                        Log.e("H4XOR", ">>>>>>>>>>>>>>>>> handler.getOriginalFile().exists() : " + handler.getOriginalFile().exists());
```

Comment:
Um... This is getting trickier by the day, what's up?

Reply:
I think CI acting up, it's fixed in next commits